src/main/java/uk/gov/hmcts/reform/sscs/constants/SscsConstants.java

```diff
@@ -38,7 +38,9 @@ private SscsConstants() {
     public static final String FIRST_NAME = "_first_name";
     public static final String LAST_NAME = "_last_name";
     public static final String ADDRESS_LINE1 = "_address_line1";
+    public static final String ADDRESS_LINE2 = "_address_line2";
     public static final String ADDRESS_LINE3 = "_address_line3";
+    public static final String ADDRESS_LINE3_COUNTY = "_address_line3";
```

Comment:
Minor thing but you can probably just use `ADDRESS_LINE3` instead of the new `ADDRESS_LINE3_COUNTY`